ajalibraries/ajantv2/src/ntv2transcode.cpp

```diff
@@ -283,7 +283,7 @@ void ConvertLinetoRGB(UByte * ycbcrBuffer,
 					  bool fUseSDMatrix,
 					  bool fUseSMPTERange)
 {
-	YCbCrAlphaPixel ycbcrPixel;
+	YCbCrAlphaPixel ycbcrPixel{};
```

Comment:
Willl this initializer work if the library is built without C++11 compiler support?